tests/user.test.js

```diff
@@ -6,18 +6,20 @@ const { user0Id, user0, setupDatabase } = require('./fixtures/db')
 beforeEach(setupDatabase)
```

Comment:
you should not setup db each time test file runs, i'd rather use `globalSetup.js` file next time

Reply:
Could you please clarify what does this mean? (see skype)